{"name": "test_search_service", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range", "trace": "self = <API_test.test_debug.TestDebug object at 0x00000206B9A56080>\n\n    def test_search_service(self):\n      for i in(1,11):\n        r=Services().search_services()\n        Services().search_termnial()\n        Services().cearte_termianl()\n        Services().search_track()\n        Services().track_upload()\n        Services().update_terminal()\n        Services().delete_terminal()\n        Services().update_service()\n        Services().delete_service()\n>       Services().search_KEY()\n\ntest_debug.py:42: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <API_test.services.Services object at 0x00000206B99160E0>\n\n    def search_KEY(self):\n        key = self.public_data()[\"params\"][\"key\"]\n        sid = self.search_services()[\"data\"][\"results\"][0][\"sid\"]\n    \n>       tid = self.search_termnial()[\"data\"][\"results\"][0][\"tid\"]\nE       IndexError: list index out of range\n\nservices.py:299: IndexError"}, "attachments": [{"name": "stdout", "source": "2c8cfe4b-cfec-4fd8-9a25-fdca4ac36385-attachment.txt", "type": "text/plain"}], "start": 1652929367837, "stop": 1652929412987, "uuid": "1fcd9704-7d0c-4484-898f-da70445c0eff", "historyId": "b51876502932a9f6041f3487ae8213f6", "testCaseId": "6c1d68848ac2647a4a71397bc8b974fb", "fullName": "test_debug.TestDebug#test_search_service", "labels": [{"name": "suite", "value": "test_debug"}, {"name": "subSuite", "value": "TestDebug"}, {"name": "host", "value": "BJI000-L26S8DL3"}, {"name": "thread", "value": "51428-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_debug"}]}